coreutils/Makefile

```diff
@@ -36,9 +36,16 @@ include ../Makefile.targ
 AUTOCONF_ENV.64 += FORCE_UNSAFE_CONFIGURE=1
 OVERRIDES.64 += FORCE_UNSAFE_CONFIGURE=1
 
+LD_LIBRARY_PATH_HACK = /tmp/extra-ldlibpath
```

Comment:
How does this get populated? Is it possible to use the proto area instead?

Reply:
See below...